Package.swift

```diff
@@ -7,7 +7,7 @@ let package = Package(
         .library(name: "APIKit", targets: ["APIKit"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/antitypical/Result.git", from: "4.0.0"),
+        .package(url: "https://github.com/antitypical/Result.git", from: "5.0.0"),
```

Comment:
There is no Result 5.0.0 version. Why is this not removed?

Reply:
this is a mistake.